Sources/FoundationEssentials/AttributedString/AttributeScope.swift

```diff
@@ -72,7 +76,13 @@ fileprivate struct LoadedScopeCache : Sendable {
             }
         }
         
-        let type = _typeByName(name) as? any AttributeScope.Type
+        guard let handle = dlopen(path, RTLD_NOLOAD),
```

Comment:
This doesn't work for Windows.  Could you please add a conditional path for this?  This is regressing the Windows build.

Reply:
This is inside `#if FOUNDATION_FRAMEWORK` which shouldn't be getting compiled in this package.

Reply:
Yep, since these various system frameworks don't exist on Windows this code is currently (and will always be) `#if`'ed out for windows

Reply:
I missed the `#if FOUNDATION_FRAMEWORK`.  Sorry for the noise!